src/main/java/com/thecookiezen/presentation/ClusterController.java

```diff
@@ -71,6 +71,14 @@ public String addHost(@Valid DockerHost dockerHost, @RequestParam long clusterId
         return "redirect:/clusters/all";
     }
 
+    @RequestMapping(value = "clusters/node", method = RequestMethod.DELETE)
+    public String deleteHost(@RequestParam Long clusterId, @RequestParam Long hostId) {
+        Cluster cluster = clusterRepository.getById(clusterId).get();
+        cluster.getHosts().removeIf(h -> h.getId() == hostId);
+        clusterRepository.save(cluster);
```

Comment:
\[Checkstyle\] WARNING: 'method def' child have incorrect indentation level 8, expected level should be 4\.